test/controllers/SendYourFileControllerSpec.scala

```diff
@@ -332,6 +335,64 @@ class SendYourFileControllerSpec extends SpecBase {
         }
       }
 
+      "must return NoContent when the file status is 'RejectedSDES'" in {
+
+        val mockFileDetailsConnector = mock[FileDetailsConnector]
+
+        val userAnswers = UserAnswers("Id")
+          .set(ConversationIdPage, conversationId)
+          .success
+          .value
+
+        when(mockFileDetailsConnector.getStatus(any[ConversationId]())(any[HeaderCarrier](), any[ExecutionContext]()))
+          .thenReturn(Future.successful(Some(RejectedSDES)))
+
+        val application = applicationBuilder(userAnswers = Some(userAnswers))
+          .overrides(
+            bind[FileDetailsConnector].toInstance(mockFileDetailsConnector)
+          )
+          .build()
+
+        running(application) {
+          val request = FakeRequest(GET, routes.SendYourFileController.getStatus().url)
+
+          val result = route(application, request).value
+
+          status(result) mustEqual OK
+
+          contentAsJson(result) shouldBe Json.toJson(URL(routes.FileProblemController.onPageLoad().url))
+        }
+      }
+
+      "must return NoContent when the file status is 'RejectedSDESVirus'" in {
```

Comment:
Can you change the name of this test as well please. It should be something like "must return OK and load the 'VirusFileFound' page when the file status is 'RejectedSDESVirus'"